resources/src/commands/heart.rs

```diff
@@ -0,0 +1,22 @@
+// This function will add a heart reaction to the passed in message
+// Send a Heart emoji to the channel with the specified timestamp from the message
+use slack_hook::{PayloadBuilder, Slack};
+
+// TODO Use reactions.add
+pub async fn run(channel: &str) {
+    let slack_webhook_url: String = crate::get_env_var(crate::DEVIL_BOT_TEST_CHANNEL_URL);
+    let slack = Slack::new(&*slack_webhook_url).unwrap();
+    let p = PayloadBuilder::new()
+        .text("This message should have a heart? :heart:")
+        .icon_emoji(":heart:")
```

Comment:
Probably this whole file should be `emoji_reaction.rs` and you should just pass in which emoji you want to use.

Reply:
Plan is to use reactions.add and make a reusable function for it. Will make an `emoji_reaction.rs` would you like that in this PR or should it be a new issue?

Reply:
Will address in #41